pkg/cloud/services/networking/floatingip_test.go

```diff
@@ -77,7 +76,6 @@ func Test_GetOrCreateFloatingIP(t *testing.T) {
 			tt.expect(mockClient.EXPECT())
 			s := Service{
 				client: mockClient,
-				logger: logr.DiscardLogger{},
```

Comment:
Why was this removed?

Reply:
check the commit msg  :)

the logr 0.4=>1.2 update makes this happen :)